swap/types.go

```diff
@@ -49,3 +54,120 @@ type EmitChequeMsg struct {
 type ConfirmChequeMsg struct {
 	Cheque *Cheque
 }
+
+// Uint256 represents an unsigned integer of 256 bits
+type Uint256 struct {
+	value big.Int
```

Comment:
the pointer was removed in the first place to avoid `nil` values in `Uint256` variables. this way, there's always _at least_ an underlying value of `0`.